.changeset/popular-eggs-notice.md

```diff
@@ -0,0 +1,5 @@
+---
+"@primer/view-components": minor
```

Comment:
@hectahertz I made this a `minor` change as PVC has yet to have a `major` release.